receiver/jaegerreceiver/trace_receiver.go

```diff
@@ -388,6 +407,12 @@ func (jr *jReceiver) startCollector(ctx context.Context, host component.Host) er
 			return err
 		}
 
+		jr.settings.Logger.Info("Starting HTTP server for Jaeger Collector",
```

Comment:
just had question that should this be instead something like "Starting HTTP server for Jaeger Collector (Thrift over HTTP)"  as current one doesnt tells which protocol is being used.